firebase-vertexai/firebase-vertexai.gradle.kts

```diff
@@ -63,6 +63,7 @@ dependencies {
   implementation("com.google.guava:listenablefuture:1.0")
   implementation("androidx.concurrent:concurrent-futures:1.2.0-alpha03")
   implementation("androidx.concurrent:concurrent-futures-ktx:1.2.0-alpha03")
+  api("com.google.firebase:firebase-auth-interop:18.0.0")
```

Comment:
Since we are building this SDK from scratch, we can make this an `implementation` dependency instead of `api`. We defaulted to `api` in the older SDKs since we didn't want to break people unnecessarily.